pandas/core/series.py

```diff
@@ -666,11 +666,13 @@ def __getitem__(self, key):
 
                     # we need to box if we have a non-unique index here
                     # otherwise have inline ndarray/lists
```

Comment:
can you update this comment a bit, its not longer non-unique, but getting back a scalar loc for the key, IOW its a unique key

Reply:
Is it unique key?